src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -56,6 +57,10 @@ type LSPInlayHint =
     Pos: Types.Position
     Kind: InlayHintKind }
 
+type InlayHintData =
```

Comment:
is this type still needed now that we no longer provide data to the hints?

Reply:
You're absolutely right  
-> removed